lib/drivers/medtronic/docs/packetStructure.md

```diff
@@ -2,6 +2,45 @@
 
 Below we've documented the structure of the various packets in the Medtronic pump history, matching the fields and record types to our [data model](http://developer.tidepool.io/data-model). Unknown bits are designated with `??`.
 
+## MiniMed 5/7 series Message Envelope
+![basal](images/svg/MiniMedMessage.svg)
+
+| Field     | Bytes | Data Type  |   Comments   |
+|-----------|:-----:|:----------:|--------------|
+| Header    | 2     | Bytes      | `0x5101`     |
+| Pump Serial Number | 6     | String     | ASCII numbers, always `'000000'` for 600-series |
```

Comment:
Maybe we can remove the "always `'000000'` for 600-series" bit as this is specific to 5/7-series?